src/components/TodoContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
+
+function TodoContainer() {
+  const [todos, setTodos] = useState([]);
+
+  const handleChange = (id) => {
+    setTodos(todos.map((todo) => {
+      if (todo.id === id) {
+        return {
+          ...todo,
+          completed: !todo.completed,
+        };
+      }
+      return todo;
+    }));
+  };
+
+  const handleDelete = (id) => {
+    setTodos(todos.filter((todo) => todo.id !== id));
+  };
+
+  const addTodoItem = (title) => {
+    const newTodo = {
+      id: uuidv4(),
+      title,
+      completed: false,
+    };
+    setTodos([...todos, newTodo]);
+  };
+
+  const handleTodoUpdate = (title, id) => {
+    setTodos(todos.map((todo) => {
+      if (todo.id === id) {
+        return {
+          ...todo,
+          title,
+        };
+      }
+      return todo;
+    }));
+  };
+
+  useEffect(() => {
+    fetch('https://jsonplaceholder.typicode.com/todos?_limit=10')
+      .then((response) => response.json())
+      .then((data) => setTodos(data));
+  }, []);
```

Comment:
- [ ] (Optional) I would recommened you to use local storage instead of using the remote API you can use a local browser API like `Local Storage` beacuse the objective of the project is to enable users to record  their dayly todos. so If you use local storage API it would be fast for accessing the data.